modules/test/playwright/tests/osb-faro-web/segments.spec.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import getRandomString from '../../utils/getRandomString';
+import {faroConfig} from './faro.config';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test('check if updated custom event displayName is shown on segment criteria card', async ({
+	page,
+}) => {
+	const channelName = 'My Property - ' + getRandomString();
+	const customEventName = 'CustomEvent' + new Date().getTime();
+
+	await syncAnalyticsCloud(page, channelName);
+
+	await page.goto(liferayConfig.environment.baseUrl);
+
+	await page.waitForTimeout(3000);
+
+	await page.evaluate(
+		({customEventName}) => {
+			//@ts-ignore
+			if (window.Analytics) {
+				//@ts-ignore
+				window.Analytics.track(customEventName, {
+					propDate: '2024-05-20T01:00:00.000',
+					propDuration: 66840000,
+					propNum: 18,
+					propString: 'test',
+					propBool: true,
+				});
+			}
+		},
+		{customEventName}
+	);
+
+	await page.waitForTimeout(3000);
+
+	await page.goto(faroConfig.environment.baseUrl);
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+
+	await page.locator('.channels-menu.button-root').click();
+	await page.getByRole('link', {name: channelName}).click();
+	await page.getByRole('link', {name: 'Settings'}).click();
+	await page.getByRole('link', {name: 'Definitions'}).click();
+	await page.getByRole('link', {name: 'Events'}).click();
+	await page.getByRole('link', {name: 'Custom Events'}).click();
+	await page.getByRole('link', {name: customEventName}).click();
+	await page.getByRole('button', {name: 'Edit'}).click();
```

Comment:
Hey @fmarcio, I think it would be good to search in the search field to ensure that your custom event appears on the screen before clicking. Today you only have your custom event, but in the future, you maybe have several and yours will not appear in the default pagination

Reply:
@ViniLopes20 done